test/jdk/ProblemList.txt

```diff
@@ -468,6 +468,8 @@ sun/java2d/DirectX/OnScreenRenderingResizeTest/OnScreenRenderingResizeTest.java
 java/awt/Window/8159168/SetShapeTest.java 8274106 macosx-aarch64
 java/awt/image/multiresolution/MultiResolutionJOptionPaneIconTest.java 8274106 macosx-aarch64
 
+java/awt/font/Rotate/RotatedTextTest.java 8219641 linux-all
+
```

Comment:
Looks good to me, except for its placement: shouldn't it be placed close to other font tests? Shouldn't the list be sorted?

Do we always add newly excluded tests to the end of the list?

Reply:
You are right, it all seems a bit unordered. If you like, I could place it next to line 221 or 416 where other java/awt/font tests are excluded. Not sure what the rule is but it seems like it's at least not alphabetically ordered.

Reply:
I couldn't find any best practices for problem-listing.

Yet keeping tests in the same area closer seems like a good idea to me. Either line is fine with me. Thanks.

I started an internal discussion in clientlibs to see whether there are any conventions.

Reply:
> Yet keeping tests in the same area closer seems like a good idea to me. Either line is fine with me.

Let's go for the latter one, line 416.

> I started an internal discussion in clientlibs to see whether there are any conventions.

It looks there are no strict conventions. There groups for *awt* and *swing*, there are also some subheaders to group tests which fail with the same root cause, like the one above the line you're adding.

At the same time, I still think it's better to group tests… This one is in `awt/font`, let's add close to another one.

Other than that, it looks good to me, I'll approve it.

Reply:
OK, great. I've moved the exclusion to the proposed line. Thanks for checking.

Reply:
Thank you, Christoph!